openapi/p4pa-pu-bff.openapi.yaml

```
@@ -1652,6 +1652,41 @@ paths:
         '500':
           description: Internal Server Error
       x-spring-paginated: true
+  /organization/{organizationId}/printpaymentnotice/generate/{debtPositionId}:
+    post:
```

this API wil not modify or create any data
```suggestion
    get:
```